lib/charms/istio_beacon_k8s/v0/service_mesh.py

```diff
@@ -184,7 +185,7 @@ def __init__(self, *args):
 
 LIBID = "3f40cb7e3569454a92ac2541c5ca0a0c"  # Never change this
 LIBAPI = 0
-LIBPATCH = 10
+LIBPATCH = 11
```

Comment:
Because I was a jerk and merged #119 before you merged this :)

```suggestion
LIBPATCH = 12
```

Reply:
done!